src/main/java/racingcar/view/CarRacingResultView.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.view;
+
+import java.util.List;
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.domain.Racing;
+
+public class CarRacingResultView {
+
+    static public void printCurrentRaceResult(Racing racing) {
```

Comment:
제어자 선언에 대해서도 컨벤션이 있습니다~ 확인해보시고 적용해주세요!

Reply:
조금 감이 안 오는데 아래와 같은 형태가 되어야 할까요?

```suggestion
        cars.getCarList()
            .forEach(
                car -> System.out.println(buildCurrentPositionString(car))
            );
```

Reply:
제어자 선언 컨벤션 말씀드린거였어요!
아래 컨벤션 `3.3. 제한자 선언의 순서` 보시면 됩니다!
https://naver.github.io/hackday-conventions-java/#modifier-order

```suggestion
    public static void printCurrentRaceResult(Racing racing) {
```